### ISSUE TITLE
Remove virtio-fs patches from custom branch

### ISSUE THREAD

**Issue by sboeuf** [cloud-hypervisor/cloud-hypervisor#474]
`virtio-fs` is now part of the Linux kernel 5.4. We need to try it out and validate it works without any extra patch. This way, the custom kernel will only carry `virtio-iommu` patches :)

**Comment by sboeuf** [comment]
Discussed with `virtio-fs` maintainers, but unfortunately the DAX support is not part of the merged series. So for a fully performant `virtio-fs` solution, we'll have to wait until next kernel release.
In the meantime, I still think we should bump our custom kernel to 5.4, and apply remaining patches.

**Comment by sameo** [comment]
> In the meantime, I still think we should bump our custom kernel to 5.4, and apply remaining patches.

Agreed.

**Comment by rbradford** [comment]
@sboeuf I think there are just a few patches left?

**Comment by sboeuf** [comment]
Yes there are still a few DAX specific patches left, so let's wait a few more kernel releases before we can close this issue.

**Comment by rbradford** [comment]
All virtio-fs patches are now removed from our kernel fork!
